angular/core/models/version_records_interface.coffee

```diff
@@ -1,3 +1,15 @@
 angular.module('loomioApp').factory 'VersionRecordsInterface', (BaseRecordsInterface, VersionModel) ->
   class VersionRecordsInterface extends BaseRecordsInterface
     model: VersionModel
+
+    fetchByDiscussion: (discussionKey, options = {}) ->
+      @fetch
+        params:
+          model: 'discussion'
+          discussion_id: discussionKey
+
+    fetchByComment: (comment, options = {}) ->
```

Comment:
Why pass the model on one and the key on the other? I think the convention is to pass id/key, rather than the whole object, but I'd have a poke around and go with one.